docs/README.md

```diff
@@ -32,6 +32,79 @@ $ ./build_docs.sh
 
 The site can be viewed at http://localhost:1313/
 
+## Include externally hosted documentation
+
+With the ongoing efforts to move Flink's connectors from this repository to individual, dedicated
+repositories, this also requires the documentation to be hosted outside this repo. However, 
+we still want to serve all documentation as a whole on the Flink documentation website.
+
+In order to achieve this, we're using [Hugo Modules.](https://gohugo.io/hugo-modules/configuration/) 
+to create a virtual filesystem. 
+
+Adding new externally hosted documentation requires the following steps to be taken:
+
+1. (If necessary) Move the existing documentation to the new repository
+2. In this new repository, in the `docs` folder, create a file `go.mod` containing:
+
+```go
+module github.com/apache/flink-connector-<repositoryname>/docs
+
+go 1.18
+```
+
+Replace <repositoryname> with the name of your repository.
+See https://github.com/apache/flink-connector-elasticsearch/tree/main/docs/go.mod for an example.
+3. In this new repository, in the `docs` folder, create a `config.toml` file containing:
+
+```yaml
+module:
+  mounts:
+    - source: content/docs/connectors/datastream/<filename>.md
+      target: content/docs/connectors/datastream/<filename>.md
+      lang: en
+    - source: content/docs/connectors/table/<filename>.md
+      target: content/docs/connectors/table/<filename>.md
+      lang: en
+    - source: content.zh/docs/connectors/datastream/<filename>.md
+      target: content.zh/docs/connectors/datastream/<filename>.md
+      lang: zh
+    - source: content.zh/docs/connectors/table/<filename>.md
+      target: content.zh/docs/connectors/table/<filename>.md
+      lang: zh
+```
+
+Replace <filename> with the name of the file you want to include in the documentation.
+See https://github.com/apache/flink-connector-elasticsearch/tree/main/docs/config.toml for an example.
```

Comment:
Not sure if I did something wrong, I got 404 with this link.

Reply:
That's because the ES side is also still in the draft stage. https://github.com/apache/flink-connector-elasticsearch/pull/13